e2e/fixtures/authenticatedUserPages.ts

```diff
@@ -110,8 +114,9 @@ export const testAuth = base.extend<
   // ...per worker + browserName
   // ...depends on createUsers worker fixture to create each test user
   storageStatePaths: [
+    // eslint-disable-next-line  @typescript-eslint/no-unused-vars
     async ({ browser, browserName, createUsers }, use) => {
```

Comment:
`createUsers` is not used, but must be included so that the createUser worker fixture creates each test user before this worker fixture runs. Ignore this type error.

Reply:
It's not intuitive to me why `createUsers` needs to be destructured in the signature to accomplish different behavior. Could you explain or point me to some documentation? And also leave a comment alongside the comment used to disable the rule?

Reply:
Sure! The destructured fixtures are the dependencies of the current fixture, as described in the [Execution Order docs](https://playwright.dev/docs/test-fixtures#execution-order). Since the test users must be created before the test user can be authenticated, the `storageStatePaths` worker fixture depends on the `createUsers` worker fixture. This tells Playwright to set up `createUsers` before executing `storageStatePaths`. I'll add a comment to clarify in the code!